### DISCUSSION TITLE
Tienda en Línea

### DISCUSSION THREAD

**Discussion by NeftaliYagua** [Coderic/.github#4] [Ideas]
Proyecto para tienda en línea.
Origenes:
- Shopify
- empretienda.com
- mercadoshops.com.co

**Comment by NeftaliYagua** [comment]
Problemas encontrados en empretienda.com

- Problema de autenticación
- Problema al exportar productos masivos

**Comment by alvakpo** [comment]
Problemas encontrados en Shopify
Cobra envio por articulo
No admite empresas argentinas

**Reply by alvakpo** [reply]
* no permite hacer ventas mayoristas por si tengo un proveedor que sea un mayorista
* no permite OFRECER el servicio de logistica, haciendo depender de que la empresa tenga dropshiping (envio puerta a puerta)

**Reply by alvakpo** [reply]
No incluye automatizacion en los procesos de informacion/aprobacion de temas de aduana por pais (Inteligencia artificial podria ayudar)

**Comment by Diavaaaa** [comment]
i have same problem
